components/monitoring/logging/base/configure-logging/configure-log-collectors.yaml

```diff
@@ -2,11 +2,13 @@
 apiVersion: "logging.openshift.io/v1"
 kind: "ClusterLogging"
 metadata:
+  name: "instance"
+  namespace: "openshift-logging"
```

Comment:
What determined the namespace up until now?

Reply:
@amisstea The `kustomization.yaml` file contains `namespace: "openshift-logging"`

Haven't really decided where it's better to list the namespace.

Reply:
Anyway, just to match what was already done in
`/infra-deployments/components/monitoring/logging`
I kept the namespace name just in the `kustomization.yaml` file.